viadot/sources/mindful.py

```diff
@@ -203,9 +211,9 @@ def response_to_file(
         """Save Mindful response data to file.
```

Comment:
Class method name is `response_to_file`. The docstring description is `...data to file`. The class method returns only file name  - Is it correct? 

Docstring and function name suggests there should be data in the file, but the file_name is returned.